docs/dynamic_batch_size.md

```diff
@@ -0,0 +1,195 @@
+         0 Arctic fox, white fox, Alopex lagopus 279 ; Correct match.
+Iteration 3; Processing time: 52.46 ms; speed 19.06 fps
+imagenet top results in a single batch:
+         0 bee 309 ; Correct match.
+Iteration 4; Processing time: 51.36 ms; speed 19.47 fps
+imagenet top results in a single batch:
+         0 golden retriever 207 ; Correct match.
+Iteration 5; Processing time: 46.52 ms; speed 21.50 fps
+imagenet top results in a single batch:
+         0 gorilla, Gorilla gorilla 366 ; Correct match.
+Iteration 6; Processing time: 47.18 ms; speed 21.19 fps
+imagenet top results in a single batch:
+         0 magnetic compass 635 ; Correct match.
+Iteration 7; Processing time: 48.54 ms; speed 20.60 fps
+imagenet top results in a single batch:
+         0 peacock 84 ; Correct match.
+Iteration 8; Processing time: 48.72 ms; speed 20.53 fps
+imagenet top results in a single batch:
+         0 pelican 144 ; Correct match.
+Iteration 9; Processing time: 48.77 ms; speed 20.51 fps
+imagenet top results in a single batch:
+         0 snail 113 ; Correct match.
+Iteration 10; Processing time: 45.51 ms; speed 21.97 fps
+imagenet top results in a single batch:
+         0 zebra 340 ; Correct match.
+
+processing time for all iterations
+average time: 49.80 ms; average speed: 20.08 fps
+median time: 48.00 ms; median speed: 20.83 fps
+max time: 63.00 ms; min speed: 15.87 fps
+min time: 45.00 ms; max speed: 22.22 fps
+time percentile 90: 53.10 ms; speed percentile 90: 18.83 fps
+time percentile 50: 48.00 ms; speed percentile 50: 20.83 fps
+time standard deviation: 4.85
+time variance: 23.56
+Classification accuracy: 100.00
+...
+Start processing:
+        Model name: resnet50DAG
+        Iterations: 1
+        Images numpy path: imgs.npy
+        Images in shape: (10, 3, 224, 224)
+
+Iteration 1; Processing time: 120.19 ms; speed 66.56 fps
+imagenet top results in a single batch:
+         0 airliner 404 ; Correct match.
+         1 Arctic fox, white fox, Alopex lagopus 279 ; Correct match.
+         2 bee 309 ; Correct match.
+         3 golden retriever 207 ; Correct match.
+         4 gorilla, Gorilla gorilla 366 ; Correct match.
+         5 magnetic compass 635 ; Correct match.
+         6 peacock 84 ; Correct match.
+         7 pelican 144 ; Correct match.
+
+processing time for all iterations
+average time: 120.00 ms; average speed: 66.67 fps
+median time: 120.00 ms; median speed: 66.67 fps
+max time: 120.00 ms; min speed: 66.67 fps
+min time: 120.00 ms; max speed: 66.67 fps
+time percentile 90: 120.00 ms; speed percentile 90: 66.67 fps
+time percentile 50: 120.00 ms; speed percentile 50: 66.67 fps
+time standard deviation: 0.00
+time variance: 0.00
+Classification accuracy: 100.00
+```
+Each iteration presents the results of each infer request and details for each image in batch.
+
+With this feature we were able to successfully run two client scripts with different batch size parameters without reloading the model.
```

Comment:
It should be mentioned that the clients can be started simultaneously without performance impact from the model reloading